app/views/search/_delay_modal.html.erb

```diff
@@ -0,0 +1,24 @@
+<!--begin modal-->
+<div class="modal" id="delayWarningModal" tabindex="-1" role="dialog" aria-labelledby="delayWarningModal">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <a href="#" class="close modal-close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></a>
+        <h4 class="modal-title" id="delayWarningModalLabel">Possible Delay Ahead</h4>
+      </div>
+      <div class="modal-body">
+        <p>
+          This collection is known to cause delays and may take longer to download than expected.
+        </p>
+        <p>
+          <span id='delayOptionalMessage'></span>
```

Comment:
What happens if there is no optional message from the provider?

Reply:
@mreese84 - in that case, then span would be empty of all text and invisible to the user.  It only becomes populated with text in the event that there is an optional message to present.